### PR TITLE
vuepress-docs

### PR DESCRIPTION
**Pull request by Rahul139-stack**
Please the review the code and approved it

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+APPLICATION_ID=3HSP2MEZNG
```
Comment (skumar84pal) [inline .env:2]: It would be more clear if we mention for which purpose these constants are.

Comment (skumar84pal) [inline .env:2]: It would be more clear if we mention for which purpose these constants are.

File: README.md [hunk 2]
```diff
@@ -1 +1 @@
-# typerefinery.ai
\ No newline at end of file
+# vuepress1
```
Comment (skumar84pal) [inline README.md]: no need to change it's name

Comment (skumar84pal) [inline README.md]: no need to change it's name

**Review by skumar84pal — CHANGES_REQUESTED** [review]

**Review by wildone — CHANGES_REQUESTED** [review]
can you please add `dist` folder to ignore.

**Review by skumar84pal — APPROVED** [review]

**Review by skumar84pal — APPROVED** [review]

**Review by wildone — APPROVED** [review]

**Pull request merged**